templates/appliance/plex/raspberry-pi.html

```diff
@@ -0,0 +1,129 @@
+{% extends "appliance/_base-appliance.html" %}
+
+{% block title %}Install {{ appliance.name }} on a Raspberry Pi - Ubuntu Appliance{% endblock title %}
+
+{% block meta_description %}Install and setup the {{ appliance.name }} Ubuntu appliance on the Raspberry Pi.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1H_4L-kqKx6HJ0_3krpzdDVZ24pdVm5jb0evLMT6mcuo/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+{% if appliance.iso_url.raspberrypi %}
+<meta http-equiv="refresh" content="3;url={{ appliance.iso_url.raspberrypi }}">
+{% endif %}
+
+<section class="p-strip--suru-topped" style="overflow: visible;">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      {% if appliance.iso_url.raspberrypi %}
+      <h1 class="u-no-margin--bottom">Install the Ubuntu {{ appliance.name }} appliance for Raspberry Pi</h1>
+      <p>Your download should start automatically. If it doesn&rsquo;t,
+        <a href="{{ appliance.iso_url.raspberrypi }}">download now</a>.
+      </p>
+      {% include "appliance/shared/_verify-checksums.html" %}
```

Comment:
Would it be possible for this include to use a `<br>` instead of a p and add the content inside the p above? The spacing would looks better (I'll add a screenshot in the regular comments).